todo_app/.idea/libraries/Dart_SDK.xml

```diff
@@ -0,0 +1,19 @@
+<component name="libraryTable">
```

Comment:
A .ide mappa is egy automatikusan generált mappa. Ezt is vegyük fel a gitignore-ba kérlek és ne pusholjuk. Automatikusan generált fileokat azzért nem szoktunk felpusholni, mert azt mindenkinek legenerálja a saját fejlesztőkörnyezete, így feleslegesen foglalják csak a helyet.